src/os/ui/filter/op/isfalseop.js

```diff
@@ -0,0 +1,62 @@
+goog.provide('os.ui.filter.op.IsFalse');
+
+goog.require('os.ui.filter.op.Op');
+
+
+/**
+ * A 'PropertyIsFalse' operation class.
+ *
+ * @extends {os.ui.filter.op.Op}
+ * @constructor
+ */
+os.ui.filter.op.IsFalse = function() {
+  os.ui.filter.op.IsFalse.base(this, 'constructor',
+      'And', 'is false', 'false', undefined, undefined, 'Supports false, 0, and "false"', 'span', true);
+  this.matchHint = 'false.operation';
+};
+goog.inherits(os.ui.filter.op.IsFalse, os.ui.filter.op.Op);
+
```

Comment:
Nit: for consistency, we prefer 2 newlines between top level code blocks. Unfortunately our linter does not enforce this currently (sorry).